core-ui/src/resources/Functions/FunctionCreate.js

```diff
@@ -132,8 +139,12 @@ export function FunctionCreate({
       setResource={setFunction}
       onChange={onChange}
       formElementRef={formElementRef}
-      createUrl={`/apis/serverless.kyma-project.io/v1alpha1/namespaces/${namespace}/functions`}
+      createUrl={
+        resourceUrl ||
+        `/apis/serverless.kyma-project.io/v1alpha1/namespaces/${namespace}/functions`
```

Comment:
i know we've had a discussion about this - but maybe it might be possible to cleanly move this to namespace overview? (same with deployments)
if it can't be done cleanly, then let's keep it as-is